data/input_files/tests/solid/qs_linear.lua

```diff
@@ -0,0 +1,70 @@
+-- Comparison information
+expected_x_l2norm = 2.4276439
+epsilon = 0.0001
+
+-- Simulation time parameters
+dt      = 1.0
+
+main_mesh = {
+    type = "file",
+    -- mesh file
+    mesh = "../../../meshes/beam-hex.mesh",
+    -- serial and parallel refinement levels
+    ser_ref_levels = 1,
```

Comment:
what happens if these are both non-zero?

Reply:
It first refines the mesh on processor 0 `ser_ref_levels` time and then distributes the mesh. It then refines the mesh `par_ref_levels` times in a fully distributed fashion.

Reply:
Sometimes you need to do this if you have a very coarse mesh and the number of initial elements < number of MPI ranks.

Reply:
Is this documented somewhere?